src/utils/editor/source-editor.js

```diff
@@ -70,15 +72,15 @@ export default class SourceEditor {
     }
   }
 
-  get codeMirror(): any {
+  get codeMirror(): CodeMirror {
```

Comment:
Umm, not sure if CodeMirror is a type defined somewhere. We probably need to get its types from somewhere.

Reply:
IMO we should just use `object`; defining props for third party libraries seems unnecessary.

Reply:
it is really nice to have 3rd party types because then we have better API docs :)